addon/services/apollo.ts

```diff
@@ -97,27 +112,30 @@ export default Service.extend({
    * @return {!Object}
    * @public
    */
-  clientOptions: computed(function() {
+  @computed
+  get clientOptions() {
     return {
-      link: this.get('link'),
-      cache: this.get('cache'),
+      link: get(this, 'link'),
+      cache: get(this, 'cache'),
     };
-  }),
+  }
 
-  cache: computed(function() {
+  @computed
+  get cache(): ApolloCache<any> {
     return new InMemoryCache();
-  }),
+  }
 
-  link: computed(function() {
-    let uri = this.get('apiURL');
-    let requestCredentials = this.get('requestCredentials');
-    const linkOptions = { uri, fetch };
+  @computed
```

Comment:
Same here, should we add dependent keys here? 

```suggestion
  @computed('requestCredentials', 'apiURL')
```

Reply:
If I understand the original intention correctly, these three fields are `computed` without dependent keys, so that they are only computed once and are immutable afterwards, which makes sense because you cannot change these options, after the Apollo client has been instantiated.

See https://github.com/bgentry/ember-apollo-client/issues/186.